app/views/hyrax/file_sets/media_display/_pdf.html.erb

```diff
@@ -6,6 +6,13 @@
                     class: "representative-media",
                     alt: "",
                     role: "presentation" %>
+      <% if controller_name == 'file_sets' %>
```

Comment:
Do we need logic of something akin to `can?(:download, file_set)`?

Reply:
it's not really related to ability though. maybe a method to remove this logic from the view will suffice?

Reply:
Good question, it wasn't there prior and plus a few lines above we have this:

```erb
<% if display_media_download_link?(file_set: file_set) && (Site.account.settings[:allow_downloads].nil? || Site.account.settings[:allow_downloads].to_i.nonzero?) %>
```

Reply:
If it wasn't there prior, then that's good for me!